themes/bootstrap5/templates/search/searchbox.phtml

```diff
@@ -51,7 +51,7 @@
 ?>
 <?php $tabConfig = $this->searchTabs()->getTabConfig($this->searchClassId, $this->lookfor, $this->searchIndex, $this->searchType, $hiddenFilters); ?>
 <?php if ($this->searchType == 'advanced'): ?>
-  <div class="navbar-form navbar-left flip">
+  <div class="navbar-form">
```

Comment:
I'm not sure why the flip was here in the first place or why it was removed here, but did you check whether this works right in right-to-left languages?

Reply:
Now dropdown should be better too.

Reply:
Thanks!